grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -77,9 +78,8 @@
      * work. If {@link GrpcUnsafeBufferUtil#releaseBuffer(Object, RequestContext)} is not called, the memory
      * will be leaked.
      *
-     * <p>Due to the limited lifetime of {@link RequestContext} for blocking and async clients, this option
-     * is only really useful in conjunction with streaming clients. Even when using unary methods, it is
-     * recommended to use a streaming stub for easy access to the {@link RequestContext}.
+     * <p>Note that this isn't working if the payloads are compressed or the {@link SerializationFormat} is
+     * {@link GrpcSerializationFormats#PROTO_WEB_TEXT}.
```

Comment:
Does this fail or just do nothing? If the latter, how about 'this has no effect if ..' ?

Reply:
Let me use `this has no effect if ...` phrase, thanks!